hooks/test_post_gen_project.py

```diff
@@ -48,3 +48,19 @@ def test_copy_journal_template_files_bad(
         match=errormessage,
     ):
         copy_journal_template_files(input, project_dir)
+
+
+# C1: There are bib files in the cloned directory.
+#   Expect all bib name are inserted into manuscript,
+#   and bib files are copied into manuscript's parent dir
+#   exists.
+# C2: There are no bib files in the cloned directory.
+#   Expect manuscript bib entries will not be modified,
+#   and no files are copied.
+def test_load_bib_info():
+    assert False
+
+
+# C1: The manuscript path doesn't exist. Expect file not found error.
```

Comment:
I think this behavior is being tested by multiple of your functions so we could think about making a private function that generates a uniform error message that is reused across functions.